business/checkers/common/multi_match_selector_checker.go

```diff
@@ -183,8 +201,24 @@ func buildMultipleSidecarsValidation(scs []models.IstioValidationKey) models.Ist
 	return validations
 }
 
-func getWorkloadSelectorLabels(s kubernetes.IstioObject) map[string]string {
-	ws, found := s.GetSpec()["workloadSelector"]
+func GetWorkloadSelectorLabels(s kubernetes.IstioObject) map[string]string {
+	return getLabels("workloadSelector", "labels", s)
+}
+
+func GetSelectorLabels(s kubernetes.IstioObject) map[string]string {
+	return getLabels("selector", "matchLabels", s)
+}
+
+func HasSelector(s kubernetes.IstioObject) bool {
+	return s.HasMatchLabelsSelector()
+}
+
+func HasWorkloadSelector(s kubernetes.IstioObject) bool {
+	return s.HasWorkloadSelectorLabels()
+}
+
+func getLabels(first, second string, s kubernetes.IstioObject) map[string]string {
```

Comment:
Minor one.
I have started to add more comments in the helpers methods in my PRs as well.
Basically for me, but also it helps others to get more clarity and reduce ambiguity what this helper was necessary.

If you can get a couple of comments about that "first, second" means, perhaps it helps reviewer to not enter in the implementation detail to get it.